security/secret-protection/docs/secret-protection.rst

```diff
@@ -0,0 +1,281 @@
+
+.. _secret-protection-tutorial:
+
+Tuturial: Secret Protection
+===========================
+
+Overview
+--------
+
+Confluent’s Secret Protection feature encrypts secrets in configuration
+files. Instead of storing passwords or other sensitive data as cleartext,
+Secret Protection encrypts the data within a configuration file itself.
+
+.. note::
+
+     You can either step through this tutorial, or run the `automated
```

Comment:
This link won't work without the `:devx-examples:` type of tag

Reply:
Updated